spec/controllers/api/visits_controller_spec.rb

```diff
@@ -15,6 +15,10 @@
     JSON.parse(response.body)
   }
 
+  before do
+    request.headers['Accept-Language'] = 'cy'
+  end
```

Comment:
@alan it seems a bit odd to be setting the Accept-Language to welsh since these tests are not about testing localisation, and it generally makes the tests more awkward to read for non-welsh-speakers.